src/components/cafeteria/MobileCafeteriaPage/MobileCafeteriaPage.module.scss

```
@@ -63,3 +63,34 @@
     transform: translateX(0);
   }
 }
+
+.mo-ab-test-btn {
+  width: 330px;
+  height: 46px;
+  margin-bottom: 36px;
+  border: solid 1px #f1f8ff;
+  background-color: #f1f8ff;
+  color: #175c8e;
+  border-radius: 8px;
+  font-size: 14px;
+  display: flex;
+  justify-content: center;
```

between으로 넣으면 밑에 작성되어 있는 button에 `margin-left` 없이도 구현 가능할 거 같아요!
아니면 gap을 추가하는 것도 방법일 거 같네요!

```suggestion
  justify-content: between;
```

---

space-between 말씀하시는 것 같아서 수정하고 margin-left 삭제해보았습니다~~